### DISCUSSION TITLE
It does not support Windows 10 22H2 (19045.5198)

### DISCUSSION THREAD

**Discussion by AriesPanda77** [Atlas-OS/Atlas#1305] [Help & Support]
It does not support Windows 10 22H2 (19045.5198)
https://uupdump.net/known.php?q=category:w10-22h2

**Comment by H0ggu** [comment]
Is the ame wizard showing an error? The supported versions in the playbook lists 19045 as supported
'<SupportedBuilds>
		
		<string>19045</string>
		
		<string>26100</string>'
Are you running the latest 0.4.1 playbook?

**Reply by AriesPanda77** [reply]
AtlasPlaybook_v0.4.1.apbx    
It's this one.
I clicked start processing it prompts that this system version is not supported.

**Reply by LumiFae** [reply]
Is your machine a clean install using Windows 10?
